### PR TITLE
Fix: codecov relative path for server

### PR DESCRIPTION
**Pull request by tristanhcole**
Relative file paths for backend tests code coverage is broken. This fixes that so that files in the `server` directory are viewable. See error: https://codecov.io/gh/teamsempo/SempoBlockchain/src/b8d1bafd5dcc9b05829a2b2777d7519586f9e161/server/constants.py

**Merge in #260 first**

### CODE REVIEW

File: app/Dockerfile [hunk 1]
```diff
@@ -28,6 +28,8 @@ COPY .coveragerc /src
 
 WORKDIR /
 
+RUN apt -y install curl
```
Comment (enjeyw) [inline app/Dockerfile:31]: @tristanhcole should this be here since it was added higher in 260?

Comment (tristanhcole) [reply app/Dockerfile:31]: Yeah that’s an accident since was a fork of the other branch, will fix

Comment (enjeyw) [reply app/Dockerfile]: gone and done it from github :)

**Comment by tristanhcole** [comment]
> Good to go now!

Um this should actually be a draft PR

**Comment by tristanhcole** [comment]
I still need to fix a few things

**Pull request merged**